src/ol/webgl/TileTexture.js

```diff
@@ -8,14 +8,18 @@ import ImageTile from '../ImageTile.js';
 import TileState from '../TileState.js';
 import WebGLArrayBuffer from './Buffer.js';
 import {ARRAY_BUFFER, STATIC_DRAW} from '../webgl.js';
+import {assert} from '../asserts.js';
 import {toSize} from '../size.js';
+import {validateFloat} from './Helper.js';
 
 function bindAndConfigure(gl, texture) {
   gl.bindTexture(gl.TEXTURE_2D, texture);
   gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
   gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
   gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.LINEAR);
-  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
+  if (validateFloat()) {
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
```

Comment:
I think that only a call to `gl.texParameterf()` (and not `gl.texParameteri()`) would require the `OES_texture_float_linear` extension.  Did you find otherwise?

Reply:
I simply avoided adding something which would not be used.  Your solution suggests it does not cause any problems where `OES_texture_float_linear` is not supported.